state/upgrades.go

```diff
@@ -2452,6 +2454,63 @@ func AddSubnetIdToSubnetDocs(pool *StatePool) (err error) {
 			})
 		}
 
-		return errors.Trace(st.db().RunTransaction(ops))
+		if len(ops) > 0 {
```

Comment:
drive by after seeing run transaction with no ops fail msg in controller log on ReplacePortsDocSubnetIDCIDR.